src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -300,8 +300,6 @@ static void uct_rdmacm_cm_handle_event_connect_response(struct rdma_cm_event *ev
                   ucs_sockaddr_str(remote_addr, ip_port_str,
                                    UCS_SOCKADDR_STRING_LEN));
         uct_rdmacm_cm_ep_set_failed(cep, &remote_data, status);
-        /* notify remote side about local error */
-        rdma_disconnect(cep->id);
```

Comment:
@evgeny-leksikov it's removed to prevent a second rdma_disconnect call since uct_rdmacm_cm_ep_set_failed() will invoke it?

Reply:
it's user (UCP) responsibility how to handle error - disconnect or just destroy UCT ep (including rdmacm id) if wireup is not completed

Reply:
i guess this is needed because of https://github.com/openucx/ucx/pull/5166/files#diff-36e34c1df2a6c773b9dda08dd8bf17eeL858?

Reply:
yes, now it's unified and controlled by upper layer (UCP)